skopt/acquisition.py

```diff
@@ -264,3 +267,119 @@ def gaussian_ei(X, model, y_opt=0.0, xi=0.01, return_grad=False):
         return values, grad
 
     return values
+
+
+def gaussian_entropy_search(X, model, space, n_rep_points=50,
+                            n_trial_points=200, measure="EI", n_samples=500,
+                            random_state=None):
+    """
+    Uses entropy search to optimize the acquisition function.
+
+    1. It approximates the search space by a set of representer points sampled from
+       a suitable measure such as the "probability of improvement" or
+       "expected improvement".
+
+    2. p_min at a representer point X_i is calculated by the Integral
+       p(f(x)) \prod_{j=1}^N_rep \Theta(X_j - X_i) \\
```

Comment:
opinions on using unicode characters instead of "latex in your head" for the greek symbols?